llvm/test/tools/dxil-dis/attribute-filter.ll

```diff
@@ -19,8 +19,8 @@ define float @fma2(float %0, float %1, float %2) #1 {
   ret float %5
 }
 
-; CHECK: attributes #0 = { nounwind readnone "disable-tail-calls"="false" }
+; CHECK: attributes #0 = { nounwind readnone }
```

Comment:
Should there be a test case where fp32-denorm-mode, for example, is NOT removed?

Reply:
Nice catch. Real bug found.